contents/product-engineers/activation-metrics.mdx

```diff
@@ -0,0 +1,258 @@
+---
+date: 2024-11-05
+title: How we found our activation metric (and how you can too)
+author:
+  - raquel-m-smith
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-marketing/marketing-hog.jpg
+featuredImageType: full
+category: Engineering
+tags:
+  - Product analytics
+  - Product engineers
+  - Growth
+  - Inside PostHog
+crosspost:
+  - Blog
+---
+
+Activation AKA your product's "ah-ha" moment and the second A in the [AARRR pirate funnel framework](/product-engineers/aarrr-pirate-funnel). It's the thing that lets you know if your users are going to find success with your product.
+
+In a word, it's _important_, but it's also really difficult to define.
+
+We have 8+ products and users get activated into _each_ separately. Each also has 10+ events that we could use as an activation metric. 
+
+This means we spent a lot of time figuring out how to _figure out_ what our activation metrics are. Lucky for you, we did this all inside of PostHog, so we figured we'd share it with you, so you can do the same!
+
+But before we jump in... what even _is_ activation?
+
+## What is an activation metric?
```

Comment:
We should also add subheadings or bullets on "What makes a good activation metric" and/or "what makes a bad activation metric". 

An anecdote from my time at Facebook: It needs to be easy to A/B test with. 

We were working on a slack-like product. Our data scientist figured out that a company had the highest chances of retaining if they x number of users posting in the app after 30 days. 

This was the most accurate, but we couldn't run A/b tests with it. So our DS figured out that a metric that was highly correlated with this was "14 days since the company signed up, 3 users have posted in the last 7 days"

We worked with this metric for 6 months, but still our A/B tests took too long (we had to wait about 5 weeks, since we needed 3 weeks worth of data, but needed to wait for each company to be 14 days old).

The next iteration was "8 days since the company signed up, 2 users have posted in the last 7 days). This was highly correlated with 14-day metric, which was correlated with the 7-day.

This metrics was more workable, since it meant we could run expierments for 3-4 weeks instead of 5

Reply:
I tried writing this section but it ended up being way too vague to be useful. The anecdote is a really good one though 😅